sweep/sweeper_test.go

```diff
@@ -137,7 +137,8 @@ func createSweeperTestContext(t *testing.T) *sweeperTestContext {
 		Signer: &mock.DummySigner{},
 		GenSweepScript: func() ([]byte, error) {
 			script := make([]byte, input.P2WPKHSize)
-			script[0] = outputScriptCount
+			script[0] = 0
+			script[1] = 20
```

Comment:
What's this variable below even used for? Looks to be triggering some race race condition (see my comment below in the main body)

Reply:
Ah I think it's that two goroutine are calling this function and they both increment this variable? It seems we can either remove it, make it an atomic, or slap a defer mutex here.

Reply:
removed